src/main/java/seedu/address/logic/parser/ParserUtil.java

```diff
@@ -100,30 +100,30 @@ public static Email parseEmail(String email) throws ParseException {
     }
 
     /**
-     * Parses a {@code String tag} into a {@code Tag}.
+     * Parses a {@code String studentGroup} into a {@code StudyGroup}.
      * Leading and trailing whitespaces will be trimmed.
      *
-     * @throws ParseException if the given {@code tag} is invalid.
+     * @throws ParseException if the given {@code studentGroup} is invalid.
      */
-    public static Tag parseTag(String tag) throws ParseException {
-        requireNonNull(tag);
-        String trimmedTag = tag.trim();
-        if (!Tag.isValidTagName(trimmedTag)) {
-            throw new ParseException(Tag.MESSAGE_CONSTRAINTS);
+    public static StudentGroup parseStudentGroup(String studentGroup) throws ParseException {
+        requireNonNull(studentGroup);
+        String trimmedStudentGroup = studentGroup.trim();
+        if (!StudentGroup.isValidStudentGroupName(trimmedStudentGroup)) {
+            throw new ParseException(StudentGroup.MESSAGE_CONSTRAINTS);
         }
-        return new Tag(trimmedTag);
+        return new StudentGroup(trimmedStudentGroup);
     }
 
     /**
-     * Parses {@code Collection<String> tags} into a {@code Set<Tag>}.
+     * Parses {@code Collection<String> studyGroups} into a {@code Set<StudyGroup>}.
      */
-    public static Set<Tag> parseTags(Collection<String> tags) throws ParseException {
-        requireNonNull(tags);
-        final Set<Tag> tagSet = new HashSet<>();
-        for (String tagName : tags) {
-            tagSet.add(parseTag(tagName));
+    public static Set<StudentGroup> parseStudyGroups(Collection<String> studyGroups) throws ParseException {
```

Comment:
sorry StudyGroups and studentGroups are a bit different I feel?

Reply:
I think it might be more suitable to change this to studentGroups from StudyGroups?

Reply:
Yes, accidentally typed "studyGroup".